platform/barefoot/one-image.mk

```diff
@@ -0,0 +1,11 @@
+# sonic one image installer
+
+SONIC_ONE_IMAGE = sonic-barefoot.bin
+$(SONIC_ONE_IMAGE)_MACHINE = barefoot
+$(SONIC_ONE_IMAGE)_IMAGE_TYPE = onie
+$(SONIC_ONE_IMAGE)_INSTALLS += $(BFN_PLATFORM_MODULE) 
+$(SONIC_ONE_IMAGE)_INSTALLS += $(BFN_MONTARA_PLATFORM_MODULE)
+$(SONIC_ONE_IMAGE)_INSTALLS += $(WNC_OSW1800_PLATFORM_MODULE) 
+$(SONIC_ONE_IMAGE)_INSTALLS += $(INGRASYS_S9180_32X_PLATFORM_MODULE)
```

Comment:
The last three modules must use LAZY_INSTALL like:
`$(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(BFN_MONTARA_PLATFORM_MODULE)`
`$(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(WNC_OSW1800_PLATFORM_MODULE) `
`$(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(INGRASYS_S9180_32X_PLATFORM_MODULE)`
Or you will install all platform drivers in each platform and platform driver will init failed.
You can reference PR as follows https://github.com/barefootnetworks/sonic-buildimage/pull/6